server/2.6/Dockerfile

```diff
@@ -47,6 +48,18 @@ RUN echo "$SERVER_MD5 /tmp/debezium.tar.gz" | md5sum -c - &&\
 #
 RUN chmod -R g+w,o+w $SERVER_HOME
 
+# Stage 2: Final image
+FROM registry.access.redhat.com/ubi8/openjdk-11
+
+ENV SERVER_HOME=/debezium
```

Comment:
So if there is a need to redefine `SERVER_HOME` here in this context of a multi-build step, then I'd probably advise we redefine all ENV variables to avoid any container image regression issues where someone may expect these variables to exist.

I could see situations where a container extends the default Debezium image and maybe use one of the shipped container scripts that rely on `MAVEN_REPO_CENTRAL` or maybe even the `DEBEZIUM_VERSION`.  While we don't recommend using these images for production, we should avoid regression concerns.  wdyt?

Reply:
Added environment variables